test/drawers/drawerTests.ts

```diff
@@ -20,10 +20,12 @@ class MockAnimator implements Plottable.Animators.Plot {
   }
 }
 
-class MockDrawer extends Plottable.Drawer {
-  public _drawStep(step: Plottable.Drawers.AppliedDrawStep) {
-    step.animator.animate(this.renderArea(), step.attrToAppliedProjector);
-  }
+function getMockDrawer(dataset: Plottable.Dataset) {
```

Comment:
`createMockDrawer()` seems fine to me. What do you mean by "more aligned with what you are mocking"?